actix-session/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "actix-session"
-version = "0.6.1"
+version = "0.7.0-alpha.0"
```

Comment:
please dont do this it will screw up my release scripts :)

Reply:
Amended 😛

Reply:
context: https://github.com/actix/actix-web/blob/master/scripts/bump#L34